packages/imperative/src/cmd/__tests__/CommandProcessor.unit.test.ts

```diff
@@ -191,6 +191,61 @@ const FAKE_HELP_GENERATOR: IHelpGenerator = {
 
 const ENV_VAR_PREFIX: string = "UNIT_TEST";
 
+describe("Command Processor with --help and --version flags", () => {
```

Comment:
Typically we have only one `describe` statement at top level of a unit test file.

Could these tests be moved inside the `describe("Command Processor", ...` block?

Also please fix the failing unit tests 😋